oslabs-beta/Kubiquity
# Fix table and other stray changes

WHAT DOES THIS PR DO? 
1. Adds in a new frontend component
2. Connects new frontend component to backend

HOW TO TEST THIS PR:
1. Run `gh pr checkout 20 && npm install && npm run dev`
2. Verify that everything loads and the styling looks good (enough). 
3. Verify that the logs are coming in with the most recent log at the top of the table. 
4. Verify any logs without a "Normal" type have error styling. 
5. Verify that the BarGraph is now above the table. 
6. Look over the subheader copy and provide feedback. Are my descriptions correct??
